include/openssl/ssl.h

```diff
@@ -2445,7 +2474,7 @@ void ERR_load_SSL_strings(void);
 # define SSL_R_WRONG_SSL_VERSION                          266
 # define SSL_R_WRONG_VERSION_NUMBER                       267
 # define SSL_R_X509_LIB                                   268
-# define SSL_R_X509_VERIFICATION_SETUP_PROBLEMS           269
+# define SSL_R_X509_VERIFICATION_SETUP_PROBLEMS           101
```

Comment:
It makes it more difficult to cherry-pick to 1.0.2 from master if there are lots of these changes - and there is no benefit to this one. It should be removed.